src/ocamlorg_web/lib/handler.ml

```diff
@@ -42,12 +42,11 @@ let learn_guides req =
   Dream.redirect req (Url.tutorial (List.hd tutorials).slug)
 
 let platform _req =
-  let tools = Data.Tool.all in
   let tutorials =
     Data.Tutorial.all
     |> List.filter (fun (t : Data.Tutorial.t) -> t.section = Platform)
   in
-  Dream.html (Ocamlorg_frontend.platform ~tutorials tools)
+  Dream.redirect _req (Url.tutorial (List.hd tutorials).slug)
```

Comment:
We cannot redirect to the platform tutorials here. The platform tools page is a proper page that has its own merit and is in the correct location.

What you can do is add an entry to the sidebar that says "Overview" and links to the platform page.

Reply:
Ah ok,
I will revert changes.

What I had done so far was to add a markdown entry for the content on the platform page. 

And was currently trying to see how I can access/translate the tools.yml file to json or a simple array or something along those lines to be able to display the parts of the original platform page that was manipulating that data

Reply:
Indeed, converting the platform tools page to markdown would go too far at this point. It's likely the design on that page will be extended beyond what markdown can do.

Reply:
good point. 
Will change tact then to what you've suggested.
Thank you.